x/clock/types/codec.go

```diff
@@ -30,13 +29,18 @@ func init() {
 
 // RegisterLegacyAminoCodec registers concrete types on the LegacyAmino codec
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterConcrete(Params{}, "juno/x/clock/Params", nil)
-	legacy.RegisterAminoMsg(cdc, &MsgUpdateParams{}, "juno/x/clock/MsgUpdateParams")
+	cdc.RegisterConcrete(&MsgRegisterClockContract{}, "juno/x/clock/MsgRegisterClockContract", nil)
```

Comment:
these amino names are likely getting close to the limit. maybe change the namespace to just `juno/MsgRegisterClockContract` or `clock/MsgRegisterClockContract`

could be wrong but just want to call it out

Reply:
Sounds good, prefixed with clock/